src/OrbitGl/TimelineUi.cpp

```diff
@@ -76,10 +76,12 @@ void TimelineUi::RenderLabel(Batcher& batcher, TextRenderer& text_renderer, uint
                          TextRenderer::HAlign::Left, TextRenderer::VAlign::Middle},
                         /*out_text_pos=*/&pos, /*out_text_size=*/&size);
 
-  // Box behind the label to hide the ticks behind it.
-  size[0] += 2.f * kLabelsPadding;
+  // Box behind the label to hide the ticks behind it. We make the left padding a pixel smaller, so
+  // they don't incercept with the proper tick.
```

Comment:
"incercept" -> "intersect"?. Also, what is meant by "proper tick", I'm not sure the comment conveys the intent. Maybe "so they don't intersect with their corresponding tick line."?